ecosystem/sep-0024.md

```diff
@@ -403,7 +407,12 @@ The response should be a JSON object like:
       "fee_fixed": 5,
       "fee_percent": 1,
       "min_amount": 0.1,
-      "max_amount": 1000
+      "max_amount": 1000,
+      "fields": {
+        "email_address": {
+          "description": "Your email address will be used for for transaction status updates."
```

Comment:
```suggestion
          "description": "Email address for transaction status updates."
```
Start with the name of the field, so if it is displayed alongside a field it describes the field starting with its name. Also briefer will make it easier for including in more apps.